.gitignore

```diff
@@ -4,6 +4,8 @@
 # MacOS
 /.DS_Store
 
+# Data root
+./data_root
```

Comment:
I think it doesn't work this way - git didn't ignore this folder. But that's a detail.